compiler/src/main/scala/org/scalajs/core/compiler/GenJSExports.scala

```diff
@@ -627,6 +618,29 @@ trait GenJSExports extends SubComponent { self: GenJSCode =>
       }
     }
 
+    private def reportCannotDisambiguateError(alts: List[Exported]): Unit = {
+      val currentClass = currentClassSym.get
+
+      // Find a position that is in the current class for decent error reporting
+      val pos = alts.collectFirst {
+        case ExportedSymbol(sym) if sym.owner == currentClass => sym.pos
+        case alt: ExportedBody                                => alt.pos
+      }.getOrElse {
+        currentClass.pos
+      }
```

Comment:
I found one:
```scala
@js.native
@JSGlobal
class Foo extends js.Object {
  def foo(x: Int): Int = js.native

  @JSName("foo")
  def bar(x: Int): Int = js.native
}

@ScalaJSDefined
class Bar extends Foo {
  def foo(): Int = 42
}
```
results in
```
[error] /localhome/doeraene/projects/scalajs/examples/helloworld/HelloWorld.scala:21: Cannot disambiguate overloads for method bar with types
[error]   (x: Int)Int
[error]   (x: Int)Int
[error] class Bar extends Foo {
[error]       ^
[error] one error found
```
thanks to the code in the `getOrElse`.

I'll add that as a test.